### PR TITLE
Consider move in for PH in 1b

### PR DESCRIPTION
**Pull request by awisema**

### CODE REVIEW

File: app/models/report_generators/system_performance/fy2019/measure_one.rb [hunk 1]
```diff
@@ -516,8 +516,12 @@ def filter_days_for_homelessness dates, project_types, stop_project_types
       (! stop_project_types.include?(night[:project_type]))
     end
 
-    private def in_stop_project_on?(night, date, stop_project_types)
-      (stop_project_types.include?(night[:project_type]) && (night[:MoveInDate].blank? || night[:MoveInDate] <= date))
+    private def in_stop_project_on?(night, date, stop_project_types, consider_move_in_dates)
+      if consider_move_in_dates && PH.include?(night[:project_type])
+        return (stop_project_types.include?(night[:project_type]) && (night[:MoveInDate].present? && night[:MoveInDate] <= date))
+      else
+        return (stop_project_types.include?(night[:project_type]) && (night[:MoveInDate].blank? || night[:MoveInDate] <= date))
```
Comment (eanders) [inline app/models/report_generators/system_performance/fy2019/measure_one.rb:523]: These should be  `night[:MoveInDate] < date`

**Review by eanders — COMMENTED** [review]
Let's check and update any other SPM questions to match.

File: app/models/report_generators/system_performance/fy2019/measure_one.rb (hunk 1 above)
Comment (awisema) [reply app/models/report_generators/system_performance/fy2019/measure_one.rb:523]: I think I was wrong before...I think `<=` is correct, that way it return returns true on the move in date.

**Comment by awisema** [comment]
Measures 6 and 7 should  also consider move-in dates.
All except 6c.2 had code to handle them. Added code for 6c.2

**Pull request merged**